shop-service/src/functions/addProduct/mock.json

```diff
@@ -0,0 +1,11 @@
+{
+  "headers": {
+    "Content-Type": "application/json"
+  },
+  "body": {
+    "title": "Custom  knife",
+    "description": "Steel custom made knife according to your design",
+    "price": 100,
+    "count": 1
+  }
+}
```

Comment:
Do not leave mocks if they don't need anymore

Reply:
guess are used for autoswagger

Reply:
Swagger looks like not working 
![image](https://user-images.githubusercontent.com/9985880/176232333-4d639895-1219-4fe3-804b-f26e777125e3.png)

Reply:
strange, thanks for pointing out will check it out